cli/args/mod.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
```

Comment:
I'll create a new struct here in a future PR that will have the config file and flags in it.